assets/global.js

```diff
@@ -454,6 +454,7 @@ class ModalDialog extends HTMLElement {
     this.setAttribute('open', '');
     if (popup) popup.loadContent();
     trapFocus(this, this.querySelector('[role="dialog"]'));
+    window.pauseAllMedia();
```

Comment:
Good point.

My assumption was that modals would **always** cover the content on the screen, so that's why it wouldn't make sense to keep media in the background being played. Because elements outside the modal are not accessible (focus trap + background overlay) unless you dismiss it, I couldn't think of a scenario where pausing this content wouldn't be desired. But yeah, there might be.

I do agree with you that there's opportunity to make this class more generic and this is a really good point to validate in a refactor. For the uses we have now, I think it's okay to make it part of the general `ModalDialog` element.